timer-selector.css

```diff
@@ -31,6 +31,10 @@ fieldset {
   font-size: 0.8em;
 }
 
+#custom-css,#template,#output {
+  width: 79.2%;
```

Comment:
Why 79.2%? Seems like an unusually specific value.

Reply:
Let me check...

Reply:
Let me check this CSS code

Reply:
I seem to remember.This value was previously found to be similar to the length of the input field above. 

Of course, if there's something that looks better, I'll try it.

Reply:
The width of those input field is 50%, offset from the edge by the text before it. I can't say this specific value would work for all screen size.